pkgs/applications/graphics/freecad/default.nix

```diff
@@ -56,6 +57,11 @@ mkDerivation rec {
     hash = "sha256-XZ+fRl3CPCIFu3nHeMTLibwwFBlG/cWpKJlI58hTAuU=";
   };
 
+  postPatch = ''
```

Comment:
Is this needed? The program is not able to search the PATH for dwg2dxf?

Reply:
It is not necessary but without this, freecad is not reproducible. If one installs older or newer version of dwg2dxf, freecad may behave differently. [Nixos title page](https://nixos.org/) says: "Nix ensures that installing or upgrading one package cannot break other packages" and this is not true if this patching is removed.

An alternative way to achieving reproducibility is to add a wrapper, which prepends nix store path to dwg2dxf to PATH. I didn't know how to do it, but it seems it is documented [here](https://nixos.org/manual/nixpkgs/unstable/#qt-runtime-dependencies). It's probably cleaner than patching in absolute path, so I'll update the PR.

Reply:
Sandboxing. Yeah, makes sense. Indeed I would suggest wrapProgram - or the QT equivalent, as you did.

Reply:
PR updated.